modules/imgproc/src/contours_new.cpp

```diff
@@ -510,36 +503,33 @@ bool ContourScanner_::contourScan(const int prev, int &p, Point &last_pos, const
     return true;
 }
 
-int ContourScanner_::findFirstBoundingContour(const Point &last_pos, const int y, const int lval, int par)
+int ContourScanner_::findFirstBoundingContour(const Point& last_pos,
+                                              const int y,
+                                              const int lval,
+                                              int par)
 {
     const Point end_point(last_pos.x, y);
     int res = par;
     int cur = ctable[lval];
     while (cur != -1)
     {
-        CNode & cur_elem = tree.elem(cur);
-        if (((last_pos.x - cur_elem.body.brect.x) < cur_elem.body.brect.width) &&
-            ((last_pos.y - cur_elem.body.brect.y) < cur_elem.body.brect.height))
+        CNode& cur_elem = tree.elem(cur);
+        if (((last_pos.x - cur_elem.body.brect.x) < cur_elem.body.brect.width)
+            && ((last_pos.y - cur_elem.body.brect.y) < cur_elem.body.brect.height))
```

Comment:
previous variable is more readable.

Reply:
In this case I decided to move this condition to separate variable, then it would be aligned according to `BreakBeforeBinaryOperators` and `AlignOperands` rules:
```.cpp
        const bool isBefore = ((last_pos.x - cur_elem.body.brect.x) < cur_elem.body.brect.width)
                           && ((last_pos.y - cur_elem.body.brect.y) < cur_elem.body.brect.height);
        if (isBefore)
        {
```